span.go

```diff
@@ -8,39 +8,52 @@ package datadog
 import (
 	"encoding/binary"
 	"fmt"
+	"net/http"
 	"strconv"
 
 	"go.opencensus.io/trace"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 )
 
-// canonicalCodes maps (*trace.SpanData).Status.Code to their description. See:
+// statusCodes maps (*trace.SpanData).Status.Code to their message and http code. See:
 // https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto.
-var canonicalCodes = [...]string{
-	"ok",
-	"cancelled",
-	"unknown",
-	"invalid_argument",
-	"deadline_exceeded",
-	"not_found",
-	"already_exists",
-	"permission_denied",
-	"resource_exhausted",
-	"failed_precondition",
-	"aborted",
-	"out_of_range",
-	"unimplemented",
-	"internal",
-	"unavailable",
-	"data_loss",
-	"unauthenticated",
+var statusCodes = map[int32]statusCode{
+	trace.StatusCodeOK:                 {message: "ok", httpCode: http.StatusOK},
```

Comment:
Should we capitalize all the messages? I'm thinking to keep consistent with: https://github.com/census-instrumentation/opencensus-go/blob/b4a14686f0a98096416fe1b4cb848e384fb2b22b/plugin/ochttp/trace.go#L212-L228

Reply:
sounds good!

do you think that statusCodes is a good name?
the proto file says: `The canonical error codes for Google APIs.`